src/main/java/seedu/exercise/model/exercise/ValidationRegex.java

```diff
@@ -0,0 +1,25 @@
+package seedu.exercise.model.exercise;
+
+/**
+ * Encapsulates the different validation regex that help to check if an input is correct.
+ */
+public enum ValidationRegex {
+    ONLY_NUMBERS("\\d+(\\.\\d+)?"),
+    ONLY_ALPHABETS_AND_SPACE("^[ A-Za-z]+$"),
+    ONLY_ALPHABETS("^[A-Za-z]+$");
+
+    private final String validationRegex;
+
+    ValidationRegex(String validationRegex) {
+        this.validationRegex = validationRegex;
+    }
+
+    /**
+     * Returns the string representation of the {@code ValidationRegex} instance.
+     *
+     * @return a validation regex that corresponds to the instance invoking this method
+     */
+    public String getRegex() {
+        return this.validationRegex;
+    }
+}
```

Comment:
Can't say i agree with the use of `enum` since the regex is not being passed around or needs to be represented as objects. We just a central place to store the strings for regex. So we can just have a 

    public class ValidationRegex {
        public static final String ONLY_NUMBERS = ...
        ...
    }

This way when we need to use, we can just call `ValidationRegex.ONLY_NUMBERS` instead of needing to use `ValidationRegex.ONLY_NUMBERS.getRegex()`, The latter feels clunnky when read.

Reply:
Have made the changes 👍